onnx/shape_inference/implementation.h

```diff
@@ -33,14 +33,14 @@ class SymbolTableImpl : public SymbolTable {
  public:
   SymbolTableImpl() : index_(0) {}
 
-  void addFromGraph(const GraphProto& g) {
+  void addFromGraph(const GraphProto& g) override {
     AddExistingSymbolicDims(g.input());
     AddExistingSymbolicDims(g.output());
     AddExistingSymbolicDims(g.value_info());
   }
   // Creates a new unique symbol with the given prefix and adds it to the SymbolTable
   // Returns the newly created symbol
-  std::string createNew(const std::string& symbol_prefix = "unk__") {
+  std::string createNewImpl(const std::string& symbol_prefix) override {
```

Comment:
why not keep SymbolTable as it is except provide "unk__" as default and keep createNew name here?

Reply:
There was a discussion thread https://github.com/onnx/onnx/pull/5565#discussion_r1316524676. Does it answer some of the questions?